raiden_contracts/tests/test_channel_settle.py

```diff
@@ -273,10 +276,17 @@ def test_settle_single_direct_transfer_for_counterparty(
     )
 
     balance_proof_update_signature_B = create_balance_proof_countersignature(
-        B, channel_identifier, MessageTypeId.BALANCE_PROOF_UPDATE, *balance_proof_A
+        participant=B,
+        channel_identifier=channel_identifier,
+        msg_type=MessageTypeId.BALANCE_PROOF_UPDATE,
+        **balance_proof_A._asdict(),
     )
     token_network.functions.updateNonClosingBalanceProof(
-        channel_identifier, A, B, *balance_proof_A, balance_proof_update_signature_B
+        channel_identifier,
```

Comment:
nitpick: add kwargs

Reply:
This will be done here, after Alderaan: https://github.com/raiden-network/raiden-contracts/issues/1292

Reply:
Hmm ... that's not what I mean. I meant to specify the name of the arguments such as 

```python
token_network.functions.updateNonClosingBalanceProof(
    channel_identifier=channel_identifier,
    participant1=A,
    ....
)
```

Reply:
Positional arguments cannot come after kwargs.

Reply:
Ah I just noticed `*balance_proof_A._asdict().values()`. All right then, disregard.